cmake/Modules/FindCURL.cmake

```diff
@@ -1,19 +1,16 @@
 mark_as_advanced(CURL_LIBRARY CURL_INCLUDE_DIR)
 
-find_library(CURL_LIBRARY NAMES curl)
+find_library(CURL_LIBRARY NAMES curl libcurl)
 find_path(CURL_INCLUDE_DIR NAMES curl/curl.h)
 
-set(CURL_REQUIRED_VARS CURL_LIBRARY CURL_INCLUDE_DIR)
-
 if(WIN32)
-	find_file(CURL_DLL NAMES libcurl-4.dll
-		PATHS
-		"C:/Windows/System32"
-		DOC "Path to the cURL DLL (for installation)")
-	mark_as_advanced(CURL_DLL)
-	set(CURL_REQUIRED_VARS ${CURL_REQUIRED_VARS} CURL_DLL)
+	# If VCPKG_APPLOCAL_DEPS is ON, dll's are automatically handled by VCPKG
+	if(NOT VCPKG_APPLOCAL_DEPS)
+		find_file(CURL_DLL NAMES libcurl-4.dll libcurl.dll
+			DOC "Path to the cURL DLL (for installation)")
+		mark_as_advanced(CURL_DLL)
```

Comment:
Also supply this value to `find_package_handle_standard_args` to ensure CURL is loaded correctly without vcpkg.

Reply:
This would prevent using a static build of curl. Do you want that? 
Also, for consistency, the other FindXXX.cmake modules does not fail if the dll is not found.

Reply:
Oh, I did not see that. Thanks for pointing out.
For the consistency's sake it's fine this way.